src/ui/pages/BridgeForm/BridgeForm.tsx

```diff
@@ -433,13 +433,16 @@ function BridgeFormComponent() {
     receivePosition,
   });
 
-  const {
-    transaction: bridgeTransaction,
-    quote,
-    refetch: refetchQuotes,
-  } = quotesData;
+  const { refetch: refetchQuotes } = quotesData;
 
-  const selectedQuote = quote;
+  const defaultQuote = quotesData.quotes?.[0] ?? null;
+  // TODO: add support for quote selection, useState
+  const selectedQuote = defaultQuote;
+
+  const bridgeTransaction = useMemo(
+    () => (selectedQuote ? getQuoteTx(selectedQuote) : null),
+    [selectedQuote]
+  );
```

Comment:
I extracted this `getQuoteTx` fn, looks like it doesn't make much sense to prepare that transaction data structure inside the `useQuotes` hook, it is like a separate concern...